locksmith/src/websub/jobs/keys.ts

```diff
@@ -39,16 +40,17 @@ async function notifyHooksOfAllUnprocessedKeys(hooks: Hook[], network: number) {
       break
     }
 
-    await Promise.all(
-      hooks.map(async (hook) => {
+    await await Promise.all([
```

Comment:
Should use Promise.allSettled. Promise.all will fail if any of those fail. I don't think those tasks depend on each other so one shouldn't disrupt the other for whatever reason. WDYT?